components/DetailMovie/index.js

```diff
@@ -1,27 +1,32 @@
 import Image from "next/image";
 import { styled } from "styled-components";
+import ButtonSection from "../ButtonSection";
 
-export default function DetailGrid({ movie }) {
+export default function DetailGrid({
+  movie,
+  bookmarkedMovies,
+  setBookmarkedMovies,
+}) {
   console.log(movie);
   return (
     <StyledSection>
       <Box>
-        {movie.poster_path || movie.poster_path === null ? (
+        {movie.poster_path && movie.poster_path !== null ? (
```

Comment:
```suggestion
        {movie.poster_path !== null ? (
```
I think the second condition implies the first.